src/sas/qtgui/Calculators/media/sld_calculator_help.rst

```diff
@@ -2,57 +2,126 @@
 
 .. This is a port of the original SasView html help file to ReSTructured text
 .. by S King, ISIS, during SasView CodeCamp-III in Feb 2015.
+..
+    There is periodictable syntax for including density of components in the molecular formula field that does not appear to be implemented in SASview.
+..
+    For compounds, such as biomolecules, with exchangeable hydrogens, H[1] is used to denote the labile hydrogens. The reported contrast match point for the molecule takes into account the ratio of exchanged hydrogens.
+    This feature is not currently enabled in Sasview but is available on the NIST webpage.
 
 SLD Calculator Tool
 ===================
 
 Description
 -----------
+This tool calculates the neutron and x-ray scattering length densities (SLD) and some other useful scattering parameters of a wide range of materials including, molecules, solutions/mixtures, isotopic mixtures, and biomolecules.
+This SLD calculator utilizes the periodictable python package\ :sup:`1`, which is an extensible periodic table populated with values useful for neutron and x-ray experiments.
 
-The neutron scattering length density (SLD, $\beta_N$) is defined as
+User Inputs
+----------------------------
+**Molecular Formula** (required)
+    This field defines the material for which you are calculating the SLD. For more guidance on how to enter molecules, biomolecules, and more complex mixtures scroll down to "Specifying Materials or Mixtures in the Molecular Formula Field" section.
+
+**Mass Density** |g/cm^3| (required)
+    This field defines the density of the material for which you are calculating the SLD of your material. Density uncertainty is likely the largest source of error in the SLD calculator.
 
-.. math::
+**Neutron Wavelength** (|Ang|) (optional)
+    Wavelength is used to calculate the scattering cross-section and 1/e length. It is not required for SLD calculation. If left blank wavelength is assumed to be 6 |Ang|.
 
-  \beta_N = (b_{c1} + b_{c2} + ... + b_{cn}) / V_m
+**X-ray Wavelength** (|Ang|) (required)
+    X-ray scattering length density is energy dependent thus wavelength is a required field for x-ray SLD calculation.
+
+Calculator Output
+----------------------------
+Neutron SLD
+(|Ang^-2|)
 
-where $b_{ci}$ is the bound coherent scattering length of ith of n atoms in a molecule
-with the molecular volume $V_m$.
+X-ray SLD
+(|Ang^-2|)
 
-.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+Neutron Incoherent Cross-section (cm\ :sup:`-1`)
 
-Specifying the Compound Name
+Neutron Absorption Cross-Section (cm\ :sup:`-1`)
+
+Neutron 1/e length (cm)
+
+Specifying Materials or Mixtures in the Molecular Formula Field
 ----------------------------
+**Molecular formulas** can be entered intuitively with atoms being represented by their atomic symbol. For example, let us consider calcium carbonate (CaCO\ :sub:`3`):
```

Comment:
All instances of 'for example, let us consider' feel redundant and could probably be removed.